apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerRelationalCommandBuilderFactory.cs

```diff
@@ -0,0 +1,374 @@
+                        {
+                            case DbCommandMethod.ExecuteNonQuery:
+                            {
+                                result = dbCommand.ExecuteNonQuery();
+
+                                break;
+                            }
+                            case DbCommandMethod.ExecuteScalar:
+                            {
+                                result = dbCommand.ExecuteScalar();
+
+                                break;
+                            }
+                            case DbCommandMethod.ExecuteReader:
+                            {
+                                result
+                                    = new RelationalDataReader(
+                                        connection,
+                                        dbCommand,
+                                        dbCommand.ExecuteReader(),
+                                        commandId,
+                                        Logger);
+
+                                readerOpen = true;
+
+                                break;
+                            }
+                            default:
+                            {
+                                throw new NotSupportedException();
+                            }
+                        }
+
+                        Logger.CommandExecuted(
+                            dbCommand,
+                            executeMethod,
+                            commandId,
+                            connection.ConnectionId,
+                            result,
+                            false,
+                            startTime,
+                            stopwatch.Elapsed);
+                    }
+                    catch (Exception exception)
+                    {
+                        Logger.CommandError(
+                            dbCommand,
+                            executeMethod,
+                            commandId,
+                            connection.ConnectionId,
+                            exception,
+                            false,
+                            startTime,
+                            stopwatch.Elapsed);
+
+                        throw;
+                    }
+                    finally
+                    {
+                        if (!readerOpen)
+                        {
+                            dbCommand.Dispose();
+                            connection.Close();
+                        }
+
+                        dbCommand.Parameters.Clear();
+                    }
+
+                    return result;
+                }
+
+                protected override async Task<object> ExecuteAsync(
+                    IRelationalConnection connection,
+                    DbCommandMethod executeMethod,
+                    IReadOnlyDictionary<string, object> parameterValues,
+                    CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This can just be `= default` now.

Reply:
seems not since it needs to mimick the definition in base.  shrug.

Reply:
I don't see why that should matter. They are identical. I just tried in a test application and I was able to successfully override a method with an `= default(CancellationToken)` parameter with one that has it as `= default`.

Reply:
you are probably right.  Its most likely a bug in the resharper linter that's just making me nervous about it.  Right now, it's not a high order bit right now, but I will come back to these and switch them over time permitting.  I do appreciate the feedback.